unified-codes/.babelrc

```diff
@@ -1,4 +1,12 @@
 {
   "presets": ["@babel/preset-env"],
-  "plugins": [["@babel/transform-runtime"]]
+  "plugins": [
+    ["@babel/transform-runtime"],
+    [
+      "@babel/plugin-proposal-class-properties",
+      {
+        "loose": true
```

Comment:
results in much terser compiled code (from [babel-plugin-proposal-class-properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties))

"However, with { "loose": true }, it will compile using assignment expressions:"